conf/nginx/bedrock.conf.erb

```diff
@@ -0,0 +1,20 @@
+root /app/<%= ENV['DOCUMENT_ROOT'] %>;
+
+fastcgi_param SCRIPT_FILENAME $document_root$fastcgi_script_name;
+index index.php index.html;
+
+location / {
+    try_files $uri $uri/ /index.php?$query_string;
+}
+location ~* ^.+.(jpg|jpeg|gif|css|png|js|ico|ttf|woff)$ {
+    access_log off;
+}
+location /. {
```

Comment:
The /. should have been ~ /\. to block dotfiles. I guess as it was it blocked access to a folder named period in the root, which was dumb.